app/assets/stylesheets/application.css.scss

```diff
@@ -1,7 +1,23 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_self
+ *= require_tree .
+ */
+
 @import "bootstrap-sprockets";
 @import "bootstrap";
-
 @import url(http://fonts.googleapis.com/css?family=Open+Sans:400,700);
+
 body {
   font-family: 'Open Sans', 'sans-serif';
   background:#f0f0f0;
```

Comment:
Add your map.scss in the @import! We are a little mixed up here trying to use require and @import. do want to use sass or css? because require = css @import = scss